modules/openy_traction_rec_import/src/Commands/OpenyTractionRecImportCommands.php

```diff
@@ -109,31 +97,25 @@ public function __construct(
    */
   public function import(array $options): bool {
     if (!$this->importer->isEnabled()) {
-      $this->logger()->notice(
-        dt('Traction Rec import is not enabled!')
-      );
+      $this->logger()->notice('Traction Rec import is not enabled!');
       return FALSE;
     }
 
     if (!$this->importer->acquireLock()) {
-      $this->logger()->notice(
-        dt('Can\'t run new import, another import process already in progress.')
-      );
+      $this->logger()->notice('Can\'t run new import, another import process already in progress.');
       return FALSE;
     }
 
     if (!$this->importer->checkMigrationsStatus()) {
-      $this->logger()->notice(
-        dt('One or more migrations are still running or stuck.')
-      );
+      $this->logger()->notice('One or more migrations are still running or stuck.');
```

Comment:
Can we have an url or suggestion where to look for running or stuck migrations?

```suggestion
      $this->logger()->notice('One or more migrations are still running or stuck. Run drush ms to see status. Reset stuck migration if it is stuck.');
```